contracts/protocol/libraries/logic/SupplyLogic.sol

```diff
@@ -165,12 +165,11 @@ library SupplyLogic {
             );
         }
 
-        bool isFirstSupply = INToken(reserveCache.xTokenAddress).mint(
+        bool isFirstCollateral = INToken(reserveCache.xTokenAddress).mint(
```

Comment:
actually I think `mint` can return back:

```
(oldCollateralBalance, newCollateralBalance)
```

which can be used in any case

Reply:
```
oldCollateralBalance == 0 && newCollateralBalance > oldCollateralBalance => set true
oldCollateralBalance != 0 && newCollateralBalance == 0 => set false
```

Reply:
but obviously `isFirstCollateral` has a simpler check logic and better understanding for reading code.